.coveragerc

```diff
@@ -460,6 +460,10 @@ omit =
     homeassistant/components/frontier_silicon/browse_media.py
     homeassistant/components/frontier_silicon/media_player.py
     homeassistant/components/futurenow/light.py
+    homeassistant/components/fyta/__init__.py
```

Comment:
Which parts of this file are not covered by tests?

Reply:
fyta/__init__.py is not yet covered by tests at all.